<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommonParamsForm</class>
 <widget class="QWidget" name="CommonParamsForm">
  <property name="windowTitle">
   <string>Common Params</string>
  </property>
  <layout class="QGridLayout" name="grid">
   <!-- Dates -->
   <item row="0" column="0">
    <widget class="QLabel" name="labelStart">
     <property name="text"><string>Start date</string></property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QDateEdit" name="startDate">
     <property name="calendarPopup"><bool>true</bool></property>
     <property name="displayFormat"><string>yyyy-MM-dd</string></property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="labelEnd">
     <property name="text"><string>End date</string></property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QDateEdit" name="endDate">
     <property name="calendarPopup"><bool>true</bool></property>
     <property name="displayFormat"><string>yyyy-MM-dd</string></property>
    </widget>
   </item>

   <!-- Cloud -->
   <item row="1" column="0">
    <widget class="QLabel" name="labelCloud">
     <property name="text"><string>Max cloud (%)</string></property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="cloudSpin">
     <property name="minimum"><number>0</number></property>
     <property name="maximum"><number>100</number></property>
     <property name="value"><number>30</number></property>
    </widget>
   </item>

   <!-- Bands -->
   <item row="2" column="0">
    <widget class="QLabel" name="labelBand1">
     <property name="text"><string>Band 1</string></property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="band1Combo"/>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="labelBand2">
     <property name="text"><string>Band 2 (optional)</string></property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QComboBox" name="band2Combo"/>
   </item>

   <!-- Formula -->
   <item row="3" column="0">
    <widget class="QLabel" name="labelFormula">
     <property name="text"><string>Formula</string></property>
    </widget>
   </item>
   <item row="3" column="1" colspan="3">
    <widget class="QLineEdit" name="formulaEdit">
     <property name="placeholderText"><string>(band2-band1)/(band2+band1)</string></property>
    </widget>
   </item>

   <!-- Hint -->
   <item row="4" column="0" colspan="4">
    <widget class="QLabel" name="hintLabel">
     <property name="text">
      <string>&lt;i&gt;Tip: If Band 1 and Band 2 resolutions differ, the engine resamples; youâ€™ll see a warning.&lt;/i&gt;</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
